src/main/webapp/aboutus.html

```diff
@@ -41,17 +38,11 @@ <h2>Ivy Zhang</h2>
       <li>Aspirational Hobby: traveling around the world </li>
       <li>Ask me About: cooking</li>
     </ul>
-    <h2>Teammate C Name</h2>
-    <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
-    </ul>
-    <h2>Teammate D Name (delete if you team only has 3 students) </h2>
+    <h2>Darrell Branch</h2>
     <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
+      <li>Summer Feelz: <b><em>MOTIVATED</em></b></li>
+      <li>Aspirational Hobby: Indie Game Dev + Foodie!</li>
+      <li>Ask me About: Basketball</li>
     </ul>
   </body>
-</html>
+</html>
```

Comment:
Please add a newline at the end of the file so that the little github "missing newline" thingy goes away :)